python/src/detectors/python-zip-slip/python-zip-slip_non-compliant.py

```diff
@@ -0,0 +1,15 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# {fact rule=python-zip-slip@v1.0 defects=1}
+import zipfile
+from flask import Flask
+
+app = Flask(__name__)
+
+def non_compliant(archive_name):
+    zf = zipfile.ZipFile(archive_name)
```

Comment:
Is this applicable for all ZIP files or just the zip files coming from HTTP?
If it's for all zip files, we don't need the Flask stuff
If it's for HTTP -> It should have a decorator with proper route and `archive_name` should come from an HTTP request object